src/components/utils/blob.ts

```diff
@@ -141,3 +141,32 @@ export const isOfMimeType = (mimeTypeList: string[], blob: Blob): boolean =>
   mimeTypeList.some(
     (acceptableMimeType) => acceptableMimeType === mimeType(blob)
   )
+
+export const hmac256 = async (
+  key: string,
+  data: ArrayBuffer
+): Promise<string> => {
+  const encoder = new TextEncoder()
+  const encodedKey = encoder.encode(key)
+
+  const cryptoKey = await window.crypto.subtle.importKey(
+    'raw',
+    encodedKey,
+    {
+      name: 'HMAC',
+      hash: { name: 'SHA-256' },
+    },
+    false,
+    ['sign']
+  )
+
+  const signature = await window.crypto.subtle.sign('HMAC', cryptoKey, data)
+
+  const digest = Array.prototype.map
+    .call(new Uint8Array(signature), (byte: number) =>
+      `00${byte.toString(16)}`.slice(-2)
+    )
+    .join('')
+
+  return digest
```

Comment:
Is this supported on IE11?

Reply:
[Unfortunately not](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/subtle) :cry: I'm researching for a polyfill for IE

Reply:
Sure, but this code will be bundled in the `onfido.min.js` and without a polyfill the SDK might not load even when the feature is not used. Have you tested that the SDK loads on IE11? Btw I did a fix the other day so now you can run localhost on IE11 via `webpack-dev-server`.

Reply:
Yes, it still loads and works normally on IE11

Reply:
Great!